plugins/rplanners/ParabolicPathSmooth/DynamicPath.cpp

```diff
@@ -652,7 +652,7 @@ bool DynamicPath::TryShortcut(Real t1,Real t2,RampFeasibilityChecker& check)
     if(t1 > t2) {
         Swap(t1,t2);
     }
-    Real u1,u2;
+    Real u1 = 0.0, u2 = 0.0;
```

Comment:
@Puttichai same question here, `GetSegment()` can abort early without writing to the result, what default values would you prefer here?

Reply:
Let's assign them to `-1.0` to say they are invalid.

Reply:
done.